src/library_exceptions.js

```diff
@@ -417,11 +417,31 @@ var LibraryExceptions = {
     return Module['asm']['__cpp_exception'];
   },
 
-  // Given an WebAssembly.Exception object, returns the actual user-thrown
-  // C++ object address in the Wasm memory.
+  // Throw a WebAssembly.Exception object with the C++ tag. If traceStack is
+  // true, includes the stack traces within the exception object.
   // WebAssembly.Exception is a JS object representing a Wasm exception,
   // provided by Wasm JS API:
   // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/WebAssembly/Exception
```

Comment:
Perhaps add something like.  "In release builds this function is not needed and the native `_Unwind_RaiseException` is used instead"

Reply:
@sbc100 Added the comment.
@kripken Put the `#if ASSERTIONS` guard around it.